packages/lb-components/src/components/videoPlayer/TagToolInstanceAdaptor.tsx

```diff
@@ -0,0 +1,308 @@
+/**
+ * @file Implement TagTool interaction through class components.
+ *       Refer to: packages/lb-annotation/src/core/toolOperation/tagOperation.ts
+ * @author Glenfiddish <edwinlee0927@hotmail.com>
+ * @date 2022-05-31
+ */
+
+import React from 'react';
+import { CommonToolUtils, uuid } from '@labelbee/lb-annotation';
+import StepUtils from '@/utils/StepUtils';
+import { jsonParser } from '@/utils';
+import { VideoPlayer } from './index';
+import { VideoTagLayer } from './VideoTagLayer';
+import { IStepInfo } from '@/types/step';
+import _ from 'lodash';
+import type { ObjectString } from './types';
+import { getKeyCodeNumber } from './utils';
+
+export interface IVideoTagInstanceAdaptorProps {
+  imgIndex: number;
+  imgList: any[];
+  pageForward: () => void;
+  pageJump: (page: string) => void;
+  pageBackward: () => void;
+  onMounted: (instance: TagToolInstanceAdaptor) => void;
+  onUnmounted: () => void;
+  step: number;
+  stepList: IStepInfo[];
+}
+
+interface IVideoTagInstanceAdaptorState {
+  tagResult: any[];
```

Comment:
You can use `ITagResult` which is declared

Reply:
ITagResult has no export from lb-annotation

Reply:
Yeah, I forgot it 🥲. Maybe we need to abstract init `@types/xx` in the future